File: University/TavanDoS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            long number = scanner.nextLong();
            long pow = 1;

            while (number >= pow) {
                pow *= 2;
            }

            System.out.print(pow);
        }
    }
}

// </Ctrl+Z>

File: University/TavanDoS1.js
class Main
{
    static main(args)
    {
        try (scanner = java.util.Scanner(java.io.BufferedInputStream@b3d7190);)
        {
            var number = this.scanner.nextLong();
            var pow = 1;
            while (number >= pow)
            {
                pow *= 2;
            }
            console.log(pow);
        }
    }
}
Main.main([]);
